superset/models/annotations.py

```diff
@@ -42,6 +42,7 @@ class Annotation(Model, AuditMixinNullable):
     layer = relationship(
         AnnotationLayer,
         backref='annotation')
+    annotation_metadata = Column(Text)
```

Comment:
Knowing that this is the `Annotation` model, the pattern we've been following is to not repeat the entity name in the column names, as in just `id` and not `annotation_id` (since we know all columns in the table has to be annotation-related). There's different school of thought here and different school of data modeling will suggest different column naming schemes, but let's be consistent with other models in Superset.

Also could be nice to have `json` in the column name to let people know this column contains json text. So maybe something like `extra_json` or `json_metadata`.